homeassistant/components/zwave_js/addon.py

```diff
@@ -136,7 +137,17 @@ async def async_get_addon_discovery_info(self) -> dict:
     @api_error("Failed to get the Z-Wave JS add-on info")
     async def async_get_addon_info(self) -> AddonInfo:
         """Return and cache Z-Wave JS add-on info."""
-        addon_info: dict = await async_get_addon_info(self._hass, ADDON_SLUG)
+        addon_store_info = await async_get_addon_store_info(self._hass, ADDON_SLUG)
+        LOGGER.debug("Add-on store info: %s", addon_store_info)
+        if not addon_store_info["installed"]:
+            return AddonInfo(
+                options={},
+                state=AddonState.NOT_INSTALLED,
+                update_available=False,
+                version=addon_store_info["version"],
```

Comment:
```suggestion
                version=None,
```
`addon_store_info["version"]` will always be `None` if the addon isn't installed since it contains the installed version of the addon. We should just assume this like the others.

Reply:
Thanks!